bootstrap/example/crd.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: "apiextensions.k8s.io/v1"
 kind: "CustomResourceDefinition"
 metadata:
-  name: "mytype.example.com"
+  name: "mytypes.example.com"
```

Comment:
I realize this is the correct thing, but will this break anything?

Reply:
no, this is just the example yaml used in docs and tests